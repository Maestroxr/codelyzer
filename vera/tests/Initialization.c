

int main() 
{
	/*OK*/
	int a1 = 1;
	int a2 = 1, a3 = 1;
	/*********/	

	/*PROBLEM*/
	int x;
	int y, z;
	int a, b = 2; 
	int c = 2, d; 
	char s1, s2 = 2, s3;
	char s4 = 2,s5 , s6 = 2;
	/*********/	
	
	return 0;
	
}


