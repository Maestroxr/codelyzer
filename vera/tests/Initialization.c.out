b"Directory /task/student/sanitizer doesn't exist.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:11: Uninitialized identifier 'x' of type int.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:12: Uninitialized identifier 'z' of type int.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:12: Uninitialized identifier 'y' of type int.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:13: Uninitialized identifier 'a' of type int.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:14: Uninitialized identifier 'd' of type int.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:15: Uninitialized identifier 's3' of type char.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:15: Uninitialized identifier 's1' of type char.\r\nC:\\Work\\AutoAssessment\\Analysis Tools\\lint\\tests\\Initialization.c:16: Uninitialized identifier 's5' of type char.\r\n"